searx/settings.yml

```diff
@@ -309,6 +309,11 @@ engines:
   - name : digg
     engine : digg
     shortcut : dg
+  
+  - name: docker hub
+    engine: docker_hub
+    shortcut: dockerhub
```

Comment:
Do you mind coming up with a shorter shortcut? :D Maybe dh or drh?

Reply:
I think maintaining parity with DDG is important, and `dh` and `drh` both seem pretty unintuitive to me.